docs/operations.md

```diff
@@ -109,3 +109,23 @@ Only one Condition of type `Ready` is used.
 | 12  | Error      | Ready          | False            | ConsistencyCheckFailed | Failure during consistency check                                               |
 
 
+## Updating
+
+![Updating diagram](./assets/updating.svg)
+
+Updating logic is based on chart files placed in "ChartPath"
+
+Config Map (btp-manager-config-map)
```

Comment:
Think about different name for ConfigMap. There might be a similar named ConfigMap with configuration for btp-manager:
https://github.com/kyma-project/btp-manager/blob/main/docs/configuration.md

Reply:
how about `btp-manager-versions`?